### ISSUE TITLE
Mechanism for choosing candidates

### ISSUE THREAD

**Issue by philipjohn** [SomethingNewUK/party-constitution#3]
We need to codify the democratic mechanism for choosing candidates for any election. Things to consider;

* members in the constituency should make the decision
* there needs to be a "quorum" of local members to do so, if not perhaps fallback to authority area (for GEs this would be mean the entire membership)
* how do members put themselves forward?
* how does the vote happen?
* should there still be a vote if there is only one potential candidate? (i.e. what if that person is considered unsuitable by lots of members, should they have the chance to say they'd rather have no candidate?)

**Comment by Floppy** [comment]
Yep, we'll need to work on this so it's ready for next year's elections. I wonder if it goes in here as part of the official documents, or in the field guide, which is more general running-the-party stuff?

**Comment by philipjohn** [comment]
Hmm. In my head, this was it's natural home because it's "the rules" that govern how we do it, ensuring we're democratic and that democracy can't be overridden easily.

Granted, the FG is also subject to democratic vote so ¯\\_(ツ)_/¯

**Comment by Floppy** [comment]
Yeah, it does feel more "core". I'd be totally happy with having it in here, just asking the question :)

**Comment by philipjohn** [comment]
Combination of both perhaps. Principles of nominations, open primaries etc in here, then the mechanisms for those detailed in the FG.
